include/zephyr/display/backlight.h

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2024 Martin Kiepfer <mrmarteng@teleschirm.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Public Backlight driver APIs
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_BACKLIGHT_H_
+#define ZEPHYR_INCLUDE_DRIVERS_BACKLIGHT_H_
+
+/**
+ * @brief Backlight Driver Interface
+ * @defgroup backlight_interface Interface
+ * @ingroup io_interfaces
+ * @{
+ */
+
+#include <errno.h>
+
+#include <zephyr/types.h>
+#include <zephyr/device.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Backlight information structure
+ *
+ * This structure gathers useful information about Backlight controller.
```

Comment:
`useful information` leaves too much room for interpretation :-) Can it be more precise as to what exactly could be the use-cases of this struct